ql/processes/gjrgarchprocess.cpp

```diff
@@ -147,9 +147,9 @@ namespace QuantLib {
           //  stochastic volatility models",
           // Working Paper, Tinbergen Institute
           case PartialTruncation:
-            vol = (x0[1] > 0.0) ? std::sqrt(x0[1]) : 0.0;
-            mu =    riskFreeRate_->forwardRate(t0, t0+dt, Continuous)
-                  - dividendYield_->forwardRate(t0, t0+dt, Continuous)
+            vol = (x0[1] > 0.0) ? Real(std::sqrt(x0[1])) : 0.0;
+            mu =    riskFreeRate_->forwardRate(t0, t0+dt, Continuous).rate()
+                  - dividendYield_->forwardRate(t0, t0+dt, Continuous).rate()
```

Comment:
An alternative to this might be to have `operator+` and `operator-` for interest rates that checks for consistent conventions and returns an `InterestRate` instance, but it can be something to think about later.

Reply:
That sounds like a good idea - it might catch a few errors if conventions are not consistent in user code. At the moment this would be unnoticed. 

Should we add an issue for this idea so it doesn't get lost?

Reply:
Ok, just created that issue #1414